server/http_handler.go

```diff
@@ -161,9 +161,15 @@ func (t *tikvHandlerTool) getMvccByEncodedKey(encodedKey kv.Key) (*kvrpcpb.MvccG
 	return kvResp.MvccGetByKey, nil
 }
 
-func (t *tikvHandlerTool) getMvccByHandle(tableID, handle int64) (*kvrpcpb.MvccGetByKeyResponse, error) {
+type mvccKV struct {
+	Key   string      `json:"key"`
+	Value interface{} `json:"value"`
```

Comment:
can we use `*kvrpcpb.MvccGetByKeyResponse` as the value type?

Reply:
done.